tests/test_citations.py

```diff
@@ -245,6 +254,39 @@ def test_citation_to_citeproc_pubmed_book():
         citation_to_citeproc('pmid:29227604')
 
 
+def test_citation_to_citeproc_isbn():
+    csl_item = citation_to_citeproc('isbn:9780387950693')
+    assert csl_item['type'] == 'book'
+    assert csl_item['title'] == 'Complex Analysis'
+
+
+def test_citation_to_citeproc_isbn_title_with_quotation_mark():
+    csl_item = citation_to_citeproc('isbn:9780312353780')
+    assert csl_item['type'] == 'book'
+    assert csl_item['title'].startswith('"F" is for Fugitive')
```

Comment:
This test case is also failing.  Does it need to be `\"F\" is for fugitive`?

Reply:
I meant to be testing `get_isbn_citeproc_isbnlib`, which does produce the correct title.